src/parser/warlock/affliction/modules/talents/Deathbolt.tsx

```diff
@@ -120,13 +121,13 @@ class Deathbolt extends Analyzer {
   }
 
   get averageRemainingDotLengths() {
-    const result = {};
-    const allDurations = [];
-    Object.entries(this.remainingDotDurations).forEach(([dotId, durations]) => {
-      result[dotId] = (durations.reduce((total, current) => total + current, 0) / durations.length) || 0;
+    const result: any = {};
+    const allDurations: any = [];
+    Object.entries(this.remainingDotDurations).forEach(([dotId, durations]: any) => {
```

Comment:
After fixing the rest, you shouldnt need this `any` here. If it still complains you can give it the given types (both numbers)